app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/GoogleSignInWidget.kt

```diff
@@ -36,13 +45,34 @@ class GoogleSignInWidget : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        prefs = activity?.getSharedPreferences("LOGIN", MODE_PRIVATE)
+        editor = prefs!!.edit()
```

Comment:
you have a warning... maybe this can help? [https://stackoverflow.com/questions/42092753/sharedpreferences-edit-without-a-corresponding-commit-or-apply-call](url) .